model/src/main/proto/exploration.proto

```diff
@@ -141,7 +142,7 @@ message Hint {
 message Outcome {
   string dest_state_name = 1;
   string refresher_exploration_id = 2;
-  SubtitledHtml feedback = 3;
+  repeated SubtitledHtml feedback = 3;
```

Comment:
Is this change correct? Per https://github.com/oppia/oppia/blob/develop/core/domain/state_domain.py#L541 it seems there is just one feedback per outcome.

This might be due to the explorations you're importing being out of date.

Reply:
Is the ``correct`` change above also correct?

Reply:
Ah you're right I didn't realize the explorations were older. Reverted this and also changed the explorations structure